AxonDeepSeg/morphometrics/launch_morphometrics_computation.py

```diff
@@ -192,6 +194,18 @@ def main(argv=None):
                 # Export to csv    
                 else: 
                     pd.DataFrame(x).to_csv(current_path_target.parent / filename)
+
+                # Generate the numbers image
+                number_outfile = current_path_target.parent /(str(current_path_target.with_suffix("")) +
+                                                              "_numbers_seg.png")
```

Comment:
Would it be a better idea to add the suffix `_numbers_seg.png` in the [config](https://github.com/neuropoly/axondeepseg/blob/master/config.py) file rather than hardcoding?